main/tests/UserInterfaceTests/Ide.cs

```
@@ -141,13 +141,14 @@ public static void WaitForNoStatusMessage (string[] statusMessage, int timeoutIn
 		static void PollStatusMessage (string[] statusMessage, int timeoutInSecs, int pollStepInSecs, bool waitForMessage = true)
 		{
 			Ide.WaitUntil (() => {
+				string actualStatusMessage = null;
```

I don't think it would be a problem here. `string.Format` doesn't break on `null`.

Infact it would tell that `Workbench.GetStatusMessage` actually failed instead of getting an empty string from status bar.